app/views/datasets/show.html.erb

```diff
@@ -60,6 +60,22 @@
           <%= link_to 'Change', edit_dataset_path(@dataset.uuid, @dataset.name) %>
         </td>
     </tr>
+    <tr>
+      <tr>
+        <th class="dgu-checklist__heading">Topic</th>
+        <td>
+          <%= @dataset.topic.title if @dataset.topic %>
```

Comment:
We seem to have a mixture of `var&.field` and `var.field if var` in the code. Which one should we go with? I'm not fussed really, but we ought to be consistent.

Reply:
I'm happy to go with `&.` for new additions like this.